ssh/authorisedkeys_test.go

```diff
@@ -287,3 +290,39 @@ func (s *AuthorisedKeysKeysSuite) TestConcatAuthorisedKeys(c *gc.C) {
 		c.Check(ssh.ConcatAuthorisedKeys(test.a, test.b), gc.Equals, test.result)
 	}
 }
+
+func (s *AuthorisedKeysKeysSuite) TestAddKeysToFile(c *gc.C) {
+	key1 := sshtesting.ValidKeyOne.Key + " user@host"
+	err := ssh.AddKeysToFile(testSSHUser, alternativeKeysFile2, []string{key1})
+	c.Assert(err, jc.ErrorIsNil)
+
+	list1, err := ssh.ListKeysFromFile(testSSHUser, alternativeKeysFile2, ssh.FullKeys)
+	c.Assert(err, jc.ErrorIsNil)
+	c.Assert(list1, gc.DeepEquals, []string{key1})
+
+	key2 := sshtesting.ValidKeyTwo.Key + " user@host"
+	err = ssh.AddKeysToFile(testSSHUser, alternativeKeysFile3, []string{key2})
+	c.Assert(err, jc.ErrorIsNil)
+
+	list2, err := ssh.ListKeysFromFile(testSSHUser, alternativeKeysFile3, ssh.FullKeys)
+	c.Assert(err, jc.ErrorIsNil)
+	c.Assert(list2, gc.DeepEquals, []string{key2})
+}
+
+func (s *AuthorisedKeysKeysSuite) TestDeleteKeysFromFile(c *gc.C) {
```

Comment:
We want 2 tests - one for deleting some keys, another for deleting all keys

Reply:
Done